src/bigquery.ts

```diff
@@ -2232,6 +2232,14 @@ export class BigQuery extends Service {
         job!.getQueryResults(options, callback as QueryRowsCallback);
         return;
       }
+      // If timeout override was provided, return error.
+      if (queryReq.timeoutMs) {
+        const err = new Error(
```

Comment:
nit, does this need to be of type `GoogleError` or do we not use those here? (or is there an equivalent from nodejs-common?)

Reply:
This is replicating the same behavior of `jobs.getQueryResults`, that throws a plain Error with this message. If we change here, we should change on both places to return an `GoogleError`, but not sure if that might be considered a breaking change.

https://github.com/googleapis/nodejs-bigquery/blob/ebb3bcda5dbc316626834d1331a51ae41bbb5632/src/job.ts#L611